tests/benchmarks/test_array.py

```diff
@@ -6,7 +6,7 @@
 from ..utils import run_benchmark
 
 
-def test_quad_means(runtime):
+def test_quad_means(runtime, benchmark_time):
```

Comment:
Any test that uses this fixture should be timed, then record the result to the database.

Reply:
You actually have to pass the benchmark to the `run` caller for this to be used.